airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/DatabricksDestination.kt

```diff
@@ -85,6 +85,17 @@ class DatabricksDestination : BaseConnector(), Destination {
                 dummyNamespace,
                 dummyName
             )
+        val streamConfig =
```

Comment:
It seems like a NOP change, nothing but streamId gets used?

Reply:
yeah, this is largely a noop. The `storageOperations.writeToStage` got switched to take streamConfig instead of just streamId, so this is just to fill in that interface change

(this whole blob of code is just the `check` method - it simulates a miniature sync to verify our permissions. It really should live in the CDK, but for \<reasons\> it's just copypasted across destinations right now)

Reply:
Ah, gotcha!